### PR TITLE
Bulk Image resizer

### PR DESCRIPTION
**Pull request by Gregor-Davies**
## Description

Please describe your contribution:

A simple bulk image resizer, perfect if you are working with images and machine learning. (added because of hacktober)

<hr>
Checklist

- [x] I have added proper documentation for my contribution.
- [x] I have added myself in `contributors.json`.
- [x] My contribution isn't already present in the repository.
- [x] My contribution is working.
- [x] My changes don't generate any new warnings.
- [x] My changes don't affect any work done by other developers. 

To tick a box, replace `[ ]` with `[x]`.

### CODE REVIEW

**Comment by ashutoshkrris** [comment]
Hey @Gregor-Davies 
Could you please describe your project or code more thoroughly in the README?
You can use the code to explain how it is working with examples

**Comment by Gregor-Davies** [comment]
> Hey @Gregor-Davies Could you please describe your project or code more thoroughly in the README? You can use the code to explain how it is working with examples

Fixed with a little more info.

File: contributions/Bulk-Image-Resizer/README.MD [hunk 1]
```diff
@@ -0,0 +1,16 @@
+A simple script to bulk resize images.
```
Comment (arpit-dwivedi) [inline contributions/Bulk-Image-Resizer/README.MD:1]: Add Title using #

File: contributions/Bulk-Image-Resizer/README.MD [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+libraries used
```
Comment (arpit-dwivedi) [inline contributions/Bulk-Image-Resizer/README.MD:3]: Make it a heading or Bullet point

File: contributions/Bulk-Image-Resizer/README.MD [hunk 3]
```diff
@@ -0,0 +1,16 @@
+A simple script to bulk resize images.
+
+libraries used 
+- Pil (This is used because its quick and better than the deprecated scipy imgresize)
+- os (This is used for os related tasks)
+
+How we access the image directories:
```
Comment (arpit-dwivedi) [inline contributions/Bulk-Image-Resizer/README.MD:7]: Heading or bullet.

File: contributions/Bulk-Image-Resizer/README.MD [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+img_dir = r'Image directory'
```
Comment (arpit-dwivedi) [inline contributions/Bulk-Image-Resizer/README.MD:8]: For writing codes in the readme, [use this format](https://github.com/Learn-Write-Repeat/Open-contributions/blob/master/Markdown.md#5-coding-and-notes)

File: contributions/Bulk-Image-Resizer/README.MD [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+This is how we resize images, you can replace "80", "160" with the sizes you want.
+	img = img.resize((80, 160)
```
Comment (arpit-dwivedi) [inline contributions/Bulk-Image-Resizer/README.MD:11]: Again, For writing codes in the readme, [use this format](https://github.com/Learn-Write-Repeat/Open-contributions/blob/master/Markdown.md#5-coding-and-notes)

**Review by arpit-dwivedi — CHANGES_REQUESTED** [review]
See Inline comments

**Review by arpit-dwivedi — APPROVED** [review]
Approved

**Pull request merged**